asciinema/asciicast/raw.py

```diff
@@ -6,7 +6,7 @@
 
 
 class writer(file_writer):
-    def __init__(
+    def __init__(  # pylint: disable=too-many-arguments
```

Comment:
Btw, how are you finding these? Running `pylint` manually, or you have pylint integration in your editor?

Reply:
Both, but https://github.com/dense-analysis/ale is always running in the background.

Reply:
Got it, thanks. `ale` looks nice. Yeah, I love shellcheck, taught me better scripting practices.